pkg/plugins/trivy/plugin.go

```diff
@@ -1026,16 +1019,10 @@ func (p *plugin) getPodSpecForClientServerMode(ctx trivyoperator.PluginContext,
 			TerminationMessagePolicy: corev1.TerminationMessageFallbackToLogsOnError,
 			Env:                      env,
 			Command: []string{
-				"trivy",
+				"/bin/sh",
 			},
 			Args: []string{
-				"--quiet",
-				"image",
-				"--format",
-				"json",
-				"--server",
-				trivyServerURL,
-				optionalMirroredImage,
+				"-c", fmt.Sprintf("trivy image %s --quiet --format json --server %s > /tmp/scan/result.json &&  bzip2 -c /tmp/scan/result.json | base64", optionalMirroredImage, trivyServerURL),
```

Comment:
have the `optionalMirroredImage` and `trivyServerURL` been validated beforehand? Someone could insert arbitrary commands through those.

Reply:
@JAORMX thanks for the input , looking at it.